xclim/sdba/processing.py

```diff
@@ -173,9 +173,13 @@ def jitter_under_thresh(x: xr.DataArray, thresh: float):
 
 @parse_group
 def normalize(
-    x: xr.DataArray, *, group: Union[str, Grouper] = "time", kind: str = ADDITIVE
+    x: xr.DataArray,
+    *,
+    group: Union[str, Grouper] = "time",
+    kind: str = ADDITIVE,
+    return_norm: bool = False,
```

Comment:
Would this be cleaner to split this into two functions: `mean` and `normalize(x, group, kind, mean=None)`

Reply:
I agree. More lines to write on the user sides, but more consistent api.